mizar/dp/mizar/workflows/builtins/pods/create.py

```diff
@@ -61,25 +61,28 @@ def run(self):
         spec = {
             'hostIP': self.param.body['status']['hostIP'],
             'name': self.param.body['metadata']['name'],
-            'namespace': self.param.body['metadata']['namespace'],
-            'tenant': '',
-            # TODO (Cathy) in case of arktos
-            # get VPC and net information from annotation
-            'vpc': OBJ_DEFAULTS.default_ep_vpc,
-            'net': OBJ_DEFAULTS.default_ep_net,
+            'namespace': self.param.body['metadata'].get('namespace', 'default'),
+            'tenant': self.param.body['metadata'].get('tenant', ''),
+            'vpc': self.param.body['metadata'].get('labels', {}).get(
+                    'arktos.futurewei.com/network', OBJ_DEFAULTS.default_ep_vpc),
+            'net': self.param.body['metadata'].get('annotations', {}).get(
+                    'arktos.futurewei.com/nic', {}).get('subnet', OBJ_DEFAULTS.default_ep_net),
+            'ip': self.param.body['metadata'].get('annotations', {}).get(
+                    'arktos.futurewei.com/nic', {}).get('ip', ''),
```

Comment:
I am not sure about the correctness of the IP address here. The network operator will be allocating it. If Arktos will provide an IP for POD as a mandatory field, there must be a way to tell the network operator not to allocate the IP address. Would you please ensure that?